tower-balance/src/test.rs

```diff
@@ -1,136 +1,142 @@
-use futures::{future, Async, Poll};
-use quickcheck::*;
-use std::collections::VecDeque;
-use tower_discover::Change;
+use futures::{Async, Future};
+use tower_discover::ServiceList;
+use tower_load as load;
 use tower_service::Service;
+use tower_test::mock;
 
 use crate::*;
 
-type Error = Box<dyn std::error::Error + Send + Sync>;
+//type Error = Box<dyn std::error::Error + Send + Sync>;
 
-struct ReluctantDisco(VecDeque<Change<usize, ReluctantService>>);
-
-struct ReluctantService {
-    polls_until_ready: usize,
+macro_rules! assert_ready {
```

Comment:
TIOLI: I think the `tokio-test` crate also provides similar macros we could use here?

Reply:
citation needed

Reply:
https://github.com/tokio-rs/tokio/blob/84d5a7f5a0660dbe1e2be0e831e715f13cb59a2b/tokio-test/src/macros.rs#L3-L55

Reply:
you'd have to use `assert_ready!(svc.poll_ready())` rather than just `assert_ready!(svc)` though. also, not sure if it's worth it (it probably isn't).

Reply:
Oooooh tokio-test, not tower-test!